.circleci/config.yml

```diff
@@ -386,6 +386,7 @@ workflows:
     jobs:
       - run-all-other-tests
       - run-e2e
+      - build-docker
```

Comment:
I think we want to remove this line since the filtered version is listed below?

Reply:
ah, good call!